roles/openshift_hosted_templates/defaults/main.yml

```diff
@@ -4,7 +4,11 @@ hosted_deployment_type: "{{ 'origin' if openshift_deployment_type == 'origin' el
 
 content_version: "{{ openshift.common.examples_content_version }}"
 
-registry_url: ""
-registry_host: "{{ registry_url.split('/')[0] if '.' in registry_url.split('/')[0] else '' }}"
+openshift_hosted_images_dict:
+  origin: 'openshift/origin-${component}:${version}'
+  openshift-enterprise: 'openshift3/ose-${component}:${version}'
```

Comment:
This should be fully qualified at least for openshift-enterprise, `registry.access.redhat.com/openshift3/ose-${component}:${version}'